backend/src/main/java/com/empire/World.java

```diff
@@ -2090,7 +2083,7 @@ void evaluateGothiSpells() {
 						c.location = regions.indexOf(n.get((int)(Math.random() * n.size())));
 					}
 				}
-				for (Region r : regions) r.harvest *= .97;
+				for (Region r : regions) r.crops *= .97;
```

Comment:
I realize this is just variable name change but wanted to call out that as implemented, multiple Gothi spells active at once don't actual reduce crops [3, 6, 9, 12]%.  It's more like [3, 5.91, 8.73, 11.47] %.  The differences are fairly minute but as a player in previous games not privy to the exact code, I often tried to make calculations of what would happen next turn with different quantities and could never quite make them equal what happened according to the rules.

Reply:
The error would be further compounded if other crop-reducing/increasing effects impact a region the same turn.

Reply:
Good point! Let's open a bug.